cli/ballerina-cli/src/test/resources/test-resources/projectsForDumpGraph/package_b/modules/mod_b2/mod2.bal

```diff
@@ -0,0 +1,5 @@
+import foo/package_c.mod_c1;
+
+public function func2() {
+    mod_c1:func1();
+}
```

Comment:
this file needs a blank line at the end of the file

Reply:
fixed